src/theme/index.css

```diff
@@ -120,19 +122,19 @@ body {
 }
 
 /* Right Sidebar Freemium Container Maybe later Hover Background */
-.fefqyxr:not([data-disabled='true'])[data-hovered='true'] {
-  background: var(--primary-color) !important;
+.fefqyxr:not([data-disabled="true"])[data-hovered="true"] {
+  background: #000000 !important;
```

Comment:
Is this needed? We don't use the `#000` (all colors are at the top in the `:root`)

Reply:
Yes, we get a great contrast while using black color.